src/components/Badges/CompleteGoal.tsx

```diff
@@ -0,0 +1,42 @@
+import { IonItem, IonLabel, IonGrid, IonRow, IonCol } from '@ionic/react';
+import React from 'react';
+import './BadgeStyles.css';
+
+const GoalComplete: React.FC<{}> = () => {
+  return (
+    <IonItem>
+      <IonGrid>
+        <IonRow>
+          <IonCol>
+            <IonLabel>
+              <img
+                src="https://cdn-icons-png.flaticon.com/512/3050/3050426.png"
+                alt="goal reached!"
+                width={40}
+                height={40}
+              />
+            </IonLabel>
+          </IonCol>
+          <IonCol>
+            Summit reached
+            <IonItem class="p">
+              You set a steps goal for the event and reached it,
+              congradulations!
```

Comment:
I don't know how permanent this text is but just in case, there is a typo on 24: congratulations

Reply:
Ye, I do be making a lot of spelling errors lately